src/hooks/book/use-book-reviews.ts

```diff
@@ -57,8 +58,8 @@ export const useBookReviews = (isbn: string | undefined) => {
       if (currentUser && bookRatings.length > 0) {
         const userRatingObj = bookRatings.find(r => r.pubkey === currentUser.pubkey);
         if (userRatingObj && userRatingObj.rating !== undefined) {
-          // Store in original 0-1 scale for consistency
-          setUserRating(userRatingObj.rating);
+          // Store the raw fraction value (0-1) from the Rating object
+          setUserRating(userRatingObj.rating.fraction);
```

Comment:
After the state holds a Rating object, we can set it as a Rating directly (and the comment above can be removed)

```suggestion
          setUserRating(userRatingObj.rating);
```